### PR TITLE
Remove unused dead code and types

### PR DESCRIPTION
**Pull request by yornaath**
Have removed a lot of unused code using `yarn prunable` to analyze.
And double check if they are indeed unused by searching or finding references using ide.
Have also checked that it builds and type checks locally.

@TvrtkoM @Robiquet @robhyrk Please chime in if any of this code will be used in your future work.

### CODE REVIEW

File: components/markets/MarketCostModal.tsx [hunk 1]
```diff
@@ -1,62 +0,0 @@
...
-        {description && <div className="text-sky-600">{description}</div>}
-      </div>
-      <div className="flex flex-col items-center justify-center ml-auto">
-        <div className="font-bold font-mono">{cost}</div>
-      </div>
-    </div>
-  );
-};
-
-const MarketCostModal = ({ networkFee, permissionless, liquidity }) => {
```
Comment (Robiquet) [inline components/markets/MarketCostModal.tsx:25]: Noticed this got copy/pasted elsewhere, why not keep it in a component?

File: lib/hooks/queries/useAuthorizedReport.ts
[the remark this answers is not in the document]
Comment (TvrtkoM) [reply lib/hooks/queries/useAuthorizedReport.ts:1]: Which hooks do you think we should keep. I think we should keep only `useTimeStampForBlock` .
`useInfinitePoolsList` is buggy and simple to redo when we add new liquidity pools page.

[the remark this answers is not in the document]
Comment (yornaath) [reply lib/hooks/queries/useAuthorizedReport.ts:1]: @TvrtkoM I think useChainTime covers what useTimeStampForBlock does yeah?

[the remark this answers is not in the document]
Comment (Robiquet) [reply lib/hooks/queries/useAuthorizedReport.ts:1]: `useExtrinsicFee` and `useAuthorizedReport` look like they could be useful to me

[the remark this answers is not in the document]
Comment (TvrtkoM) [reply lib/hooks/queries/useAuthorizedReport.ts:1]: @Robiquet `useChainTime` gives timestamp for latest block and `useTimestampForBlock` can be used to get timestamp of any blocknumber in history.
But I'm ok with removing everything unused and this hooks aren't that complex anyway.

**Review by TvrtkoM — APPROVED** [review]

**Pull request merged**